src/mscorlib/corefx/System/Globalization/CultureData.cs

```diff
@@ -432,10 +436,12 @@ internal static CultureData Invariant
                     invariant._sNativeLanguage = "Invariant Language";   // Native name of this language
 
                     // Region
-                    invariant._sRegionName = "IV";                   // (RegionInfo)
-                    invariant._sEnglishCountry = "Invariant Country";    // english country name (RegionInfo)
-                    invariant._sNativeCountry = "Invariant Country";    // native country name (Windows Only)
-                    invariant._sISO3166CountryName = "IV";                   // (RegionInfo), ie: US
+                    invariant._sRegionName = "IV";                    // (RegionInfo)
+                    invariant._sEnglishCountry = "Invariant Country"; // english country name (RegionInfo)
+                    invariant._sNativeCountry = "Invariant Country";  // native country name (Windows Only)
+                    invariant._sISO3166CountryName = "IV";            // (RegionInfo), ie: US
+		            invariant._sISO3166CountryName2 = "ivc";          // 3 char ISO 3166 country name 2 2(RegionInfo)
```

Comment:
@weshaggard interesting, I am not seeing this problem in the merged file. this may be viewing problem?